src/main/java/frc/team2412/robot/subsystem/ShooterSubsystem.java

```diff
@@ -187,6 +187,10 @@ private void configMotors() {
 
     @Override
     public void periodic() {
+        if (shooterOverride) {
+            setFlywheelRPM(flywheelTestRPM);
+            setHoodAngle(hoodTestAngle);
```

Comment:
Could this section conflict with a command that also tries to set the flywheel RPM and hood angle?

Reply:
Ya that bit is intentional, in my discussions with joseph, it made sense that priority should go shooter target command -> preset -> override

Reply:
Also, since `setFlywheelRPM` and `setHoodAngle` check for `shooterOverride`, a command wouldn't be able to tell flywheel or hood to do something else during the override (unless it was using `setFlywheelVelocity` for some reason).